addon/styles/addon.scss

```diff
@@ -70,7 +70,8 @@
 }
 
 .frost-list .container {
-	border-right: 1px solid $frost-color-lgrey-1;
+  height: calc(100vh - 2px);
```

Comment:
Do we want this here or in consumers? I suppose it's a default, but 100vh is going to be too much anyway once your have a nav bar / info bar / footer present